webpack.config.js

```diff
@@ -34,14 +36,17 @@ module.exports = {
 		} ),
 		new AppCachePlugin(),
 		new HtmlWebpackPlugin( {
-			title: 'Simplenote',
-			templateContent: require( './index-builder.js' ),
-			inject: false
+			title: 'Simplenote'
 		} ),
 		new webpack.DefinePlugin( {
 			'process.env.NODE_ENV': JSON.stringify(
 				process.env.NODE_ENV || 'development'
-			)
+			),
+			config: JSON.stringify( config() )
```

Comment:
If `config` is not expected to export a function, we can use `js` or `json` or any other webpack resolvable file.

I prefer `js` because it allows for things like variables and functions and turing completeness.